freeipa/src/main/java/org/apache/directory/README.md

```diff
@@ -0,0 +1,12 @@
+This packge contains classes to aid in the generation of Kerberos Keytabs. The classes here are taken
```

Comment:
I would just drop the org/apache/directory from the package and move it to somewhere where the rest of you code lives (KeytabUtils?) You can put the readme and the classes there.

Reply:
Yes, the entire Apache DS library is a fairly heavy dependency. I've just looked into Apache Kerby which Kevin Risden mentioned to me earlier. There's a 'kerb-util' artifact that we can use which allows creating keytab. That may be more reasonable to pull in so we don't have to copy over source code. Let me try that.